requirements.dev.txt

```diff
@@ -3,7 +3,7 @@ awscli==1.29.63
 git+https://github.com/DataBiosphere/azul-chalice@1.31.2+13#egg=chalice
 coverage==6.5.0
 docker==7.1.0
-flake8==5.0.4
+flake8==7.1.1
```

Comment:
Change commit message body from 

Resolves errors originating from flake8 failing to parse f-strings

to 

Resolves errors originating from flake8 failing to parse f-strings under Python 3.12